website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -57,13 +56,16 @@ Do **not** use admin tokens for:
 Support for scoped admin tokens with more fine-grained permissions is currently in the planning stage.
 
 
-### Project level access tokens
-
-All project members can see any `CLIENT` (server-side client token) and `FRONTEND` (client-side client token) respectively.
+### API token visibility
+:::info project-level visibility
+Project-level visibility and access to API tokens was introduced in Unleash 4.22.
+:::
+By default, only admin users can create API tokens, and only admins can see their values.
+However, any [client](#client-tokens client tokens) and [front-end tokens](#front-end-tokens) that are applicable to a project, will also be visible to any members of that project that have the `READ_PROJECT_API_TOKEN` permission (all project members by default).
+Similarly, any project members with the `CREATE_PROJECT_API_TOKEN` permission can also create client and front-end tokens for that specific project ([how to create project API tokens](../how-to/how-to-create-project-api-tokens.mdx)).
 
 Use personal access tokens to:
 
-- Provide more fine-grained permissions for automation than an admin token provides
 - Give access to all project members (or anyone with `READ_PROJECT_API_TOKEN` permission) to read the token
 - Give access to the project owner (or anyone with `CREATE_PROJECT_API_TOKEN` permission) to create a token,
```

Comment:
I think we can remove this section. You need to be a project member to have the permissions listed in the last two points, so it doesn't really add any information.

```suggestion
```

Reply:
removed